x-pack/test/apm_api_integration/tests/service_groups/service_groups_api_methods.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApmApiClient } from '../../common/config';
+
+export async function getServiceGroupsApi(apmApiClient: ApmApiClient) {
+  return apmApiClient.writeUser({
+    endpoint: 'GET /internal/apm/service-groups',
+  });
+}
+
+export async function createServiceGroupApi({
+  apmApiClient,
+  serviceGroupId,
+  groupName,
+  kuery,
+  description,
+  color,
+}: {
+  apmApiClient: ApmApiClient;
+  serviceGroupId?: string;
+  groupName: string;
+  kuery: string;
+  description?: string;
+  color?: string;
```

Comment:
What about using the `ServiceGroup` type that already exist, and use a similar interface to this:

https://github.com/elastic/kibana/blob/8e12174d323cfbf8ee1e182f98ee1b86382eda4a/x-pack/plugins/apm/server/routes/service_groups/save_service_group.ts#L15-L19

```suggestion
  apmApiClient: ApmApiClient;
  serviceGroupId?: string;
  serviceGroup: ServiceGroup
```